src/NewTools-Playground-Tests/StPlaygroundPageTest.class.st

```diff
@@ -76,6 +117,53 @@ StPlaygroundPageTest >> testFromReference [
 
 ]
 
+{ #category : 'tests' }
+StPlaygroundPageTest >> testIsEmpty [ 
+	|page|
+	page := self newPage .
+	
+	self assert: page isEmpty.
```

Comment:
FYI we can use #assertEmpty: and #denyEmpty:

The advantage is that in the case the test fails on the CI, the log will display what "page" contains to help debug.

I'm suggestion one rewrite but I've seen other assertions like this in this PR

```suggestion
	self assertEmpty: page.
```